testing-guidance/workflow-tests/android/android-lkg-workflows.md

```diff
@@ -0,0 +1,77 @@
+# Android LKG Workflow Tests
+
+Testing in this area should focus on the different ways O3DE can be built and deployed into an android device and the launch of a default level successfully.
+
+## General Android Docs
+* [O3DE User Guide: Android Support for O3DE](https://www.o3de.org/docs/user-guide/platforms/android/)
+
+## Common Issues to Watch For
+
+- Binary and assets are located at their correct place for each packaging mode.
+- The most common packaging mode used during Android development is VFS (Virtual File System), be aware of network connection errors from the mobile device to Asset Processor running on PC.
+- Rendering artifacts when running default level on Android devices.
+- Low framerates when running default level on Android devices. Look out for spamming error/warnings in the Android log.
+- Asserts and Errors that appear in the Android Log while running default level.
+- Immediate crashes when launching the app on the Android device.
+- Crashes that occur after some time leaving the app running.
+
+## Workflows
+
+### Area: Build android in different packaging modes
+
+**Project Requirements**
+* Create a new default project (`DefaultProject`)
+* Modify `o3de\Registry\AssetProcessorPlatformConfig.setreg` to generate android assets in the cache. Remove `//` the following android line:
+````
+"android": "enabled",
+````
+* Make sure `DefaultProject\Platform\Android\androd_project.json` exists and that it has android settings section in it. Example:
+````
+"android_settings": {
+    "package_name": "org.o3de.DefaultProject",
+    "version_number": 1,
+    "version_name": "1.0.0",
+    "orientation": "landscape"
+},
+````
+* Build the default project on PC
+* Open editor and create a default level (`DefaultLevel`)
+* Create `DefaultProject\autoexec.cfg` file and include the following line to indicate the launcher to load the default level:
+````
+LoadLevel Levels/DefaultLevel/DefaultLevel.spawnable
+````
+* Run Asset Processor and wait until all assets have been processed.
+
+**Editor Platforms:**
+* Windows
```

Comment:
This is applicable on all host platforms, not just Windows.

Also we have instructions as part of o3de.org, that explains how to configure loading a level. Those instructions indicate how to do it using the Settings Registry (https://www.o3de.org/docs/user-guide/packaging/windows-release-builds/#set-the-starting-level)

Reply:
I'll rephrase to not mention an specific platform. 
The way that link is suggesting to set the start level is different from the one I'm using. I'll test it first to double check it's fine.
Cheers.

Reply:
I tested on Android setting the start up level like the link explains it, it worked.